extensions/src/components/extensions/tools-launcher/index.js

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
This should not be a new file, just additional component in E-Model and ME-Model.